internal/services/configstore/action/project.go

```diff
@@ -93,6 +116,8 @@ type CreateUpdateProjectRequest struct {
 	SkipSSHHostKeyCheck        bool
 	PassVarsToForkedPR         bool
 	DefaultBranch              string
+	//MembersCanPerformRunActions define if users of the organization can restart stop cancel a project run
```

Comment:
MembersCanPerformRunActions define if users of the organization can restart/stop/cancel a project run